johnsonandjohnson/Bodiless-JS
# Feat(scripts): improve npm run new to be wizard

## Changes
Switch npm run new to wizard and add options.
Update test-site git ignore & package.json
Updated doc

## Test Instructions
Precursor step:  > npm run setup

1) Run npm run new with old parameters and all should work as before:
- npm run new ../startersite 
- npm run new ../startersite --noinstall
- npm run new ../testsite test-site 
- npm run new ../testsite test-site --noinstall

2) Run npm run new with NO parameters and then you can choose what you would like to do:

3) Starter - site

> npm run new

? What type of new site do you want to create? starter

? What is location of new site? recommend with path of ../ to be outside of monorepo.: ../starter-site

? Do you want to run git-init on Destination Directory? Recommended if New Starter Site: No. & Yes

? Do you want to run npm install on Destination Directory? (skip if you plan to Pack): No & Yes

? Do you want to pack latest version of Bodiless into Destination Directory? No & Yes

? Destination directory already exists. Do you want to stop? No & Yes

4) Test - site

> npm run new

? What type of new site do you want to create? test-site

? What is location of new site? recommend with path of ../ to be outside of monorepo.: ../test-site

? Do you want to run git-init on Destination Directory? Recommended if New Starter Site: No. & Yes

? Do you want to run npm install on Destination Directory? (skip if you plan to Pack): No & Yes

? Do you want to pack latest version of Bodiless into Destination Directory? No & Yes

? Destination directory already exists. Do you want to stop? No & Yes


## Related Issues


Fixes #538
